plugins/TwoFactorAuth/SystemSettings.php

```diff
@@ -31,7 +31,7 @@ private function createRequire2FA()
     {
         return $this->makeSetting('twoFactorAuthRequired', $default = false, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
             $field->title = 'Require two-factor authentication for everyone';
-            $field->description = 'When enabled, every user has to enable two factor authentication.';
+            $field->description = 'When enabled, every user has to enable two factor authentication. Enforcing 2FA will require all users to have access to an authenticator app. This means they need to have a device like a smartphone or computer where they can install an app.';
```

Comment:
Interesting that the titles & descriptions in this file actually seems to be not translatable. Would you mind moving them to the translation file within your PR?

Reply:
I have now attempted to do that however it doesn't seem to be working as expected. Once I include a call to Piwik::translate() the general settings page still loads but only partially. The 2fa section and, it seems, everything after that is omitted.

Reply:
You need to add a `use Piwik\Piwik;` at the file head, where the other use statements are.